testing/agent/src/main/java/sbt/ForkMain.java

```diff
@@ -27,12 +27,12 @@
   // serializables
   // -----------------------------------------------------------------------------
 
-  static final class SubclassFingerscan implements SubclassFingerprint, Serializable {
+  public static final class SubclassFingerscan implements SubclassFingerprint, Serializable {
```

Comment:
I am ok with the change itself, but isn't ForkMain an implementation detail of sbt?
It's unlikely that we'd change this too much, but I don't think there's guarantee that it would remain the same either.

Reply:
Hmm, it's located in `test-agent` module and it's related to executing tests. I think it's okay that other tools are using it for running tests, but they have to take into account that it could change. Seems fair to me, wdyt?

Reply:
I guess so.